openedx/features/course_duration_limits/config.py

```diff
@@ -17,3 +24,36 @@
     flag_name=u'studio_ui',
     flag_undefined_default=False
 )
+
+EXPERIMENT_ID = 11
+EXPERIMENT_DATA_HOLDBACK_KEY = 'holdback_{0}'
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def set_value_for_content_type_gating_holdback(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
+    experiment_data_holdback_key = EXPERIMENT_DATA_HOLDBACK_KEY.format(user)
+    if event == EnrollStatusChange.enroll:
+        try:
+            ExperimentData.objects.get(
+                user=user,
+                experiment_id=EXPERIMENT_ID,
+                key=experiment_data_holdback_key,
+            )
+        except ExperimentData.DoesNotExist:
+            try:
+                content_type_gating_holdback_percentage_value = ExperimentKeyValue.objects.get(
+                    experiment_id=EXPERIMENT_ID,
+                    key="content_type_gating_holdback_percentage"
+                ).value
+                content_type_gating_holdback_percentage = float(content_type_gating_holdback_percentage_value) / 100
+            except (ExperimentKeyValue.DoesNotExist, AttributeError):
+                content_type_gating_holdback_percentage = .1
```

Comment:
Maybe just don't put people in the holdback if the percentage isn't set? That way we can release this code with no observable effect?